SPECS/sysbench/sysbench.spec

```diff
@@ -0,0 +1,166 @@
+Summary:        Scriptable database and system performance benchmark
+Name:           sysbench
+Version:        1.0.20
+Release:        1%{?dist}
+License:        GPLv2+
+Group:          Applications/System
+URL:            https://github.com/akopytov/sysbench/
+Source0:        https://github.com/akopytov/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  automake
+BuildRequires:  libaio-devel
+BuildRequires:  libtool
+BuildRequires:  make
+BuildRequires:  mariadb-devel
+BuildRequires:  pkgconfig
+BuildRequires:  postgresql-devel
+BuildRequires:  python
+Requires:       libaio
+Requires:       mariadb
+
+%description
+sysbench is a scriptable multi-threaded benchmark tool based on
+LuaJIT. It is most frequently used for database benchmarks, but can also
+be used to create arbitrarily complex workloads that do not involve a
+database server.
+
+sysbench comes with the following bundled benchmarks:
+
+- oltp_*.lua: a collection of OLTP-like database benchmarks
+- fileio: a filesystem-level benchmark
+- cpu: a simple CPU benchmark
+- memory: a memory access benchmark
+- threads: a thread-based scheduler benchmark
+- mutex: a POSIX mutex benchmark
+
+%prep
+%setup -q
+
+%build
+export CFLAGS="%{optflags}"
```

Comment:
Nit: Exporting `CFLAGS` shouldn't be necessary, as the definition is automatically inserted as part of the `%build` template.

```suggestion
```